assets/src/features/shared/hooks/useMobile.tsx

```diff
@@ -6,7 +6,7 @@ const useMobile = () => {
   const updateIsMobileState = () => {
```

Comment:
It probably was somewhere in other reviews but I think we need something more sophisticated than looking at the width pixel breakpoints

Reply:
This solution is good enough for this case because we want to detect smaller screens, not mobile devices.

Reply:
> we want to detect smaller screens, not mobile devices.

The problem is that the hook name suggests the latter. I'd suggest renaming it to better reflect the intention, e.g. useMobileViewport